meta/BRIP-0003.md

```diff
@@ -0,0 +1,150 @@
+---
+mip: 0003
+title: Stable Block Time
+author: aBear, fridrik01
+status: Draft
+created: 2025-06-16
+type: Core
+---
+
+## Summary
+
+Stable Block Time (SBT) introduces a mechanism for ensuring consistent block intervals in CometBFT-based blockchains. This enables the Berachain network to target a fixed block production interval (e.g., one block every 2 seconds), thus stabilizing BGT inflation in the absence of network partition issues.
```

Comment:
would add a small clarification in summary that this stabilizes the average block time over a large interval but doesn't lead to precise slot times as in Eth2.0

Reply:
Done